app/src/App.tsx

```diff
@@ -35,14 +117,29 @@ function App () {
         </Text>
         <IconButton
           icon={AdvancedIcon}
-          onClick={() => { }}
+          onClick={onClickSettings}
         />
       </Row>
-      <Box
-        boxType='primary'
+      <FromSection
+        getLocale={getLocale}
+        onClickPresetAmount={onClickPresetAmount}
+        onInputChange={handleOnSetFromAmount}
+        inputValue={fromAmount}
+        onClickSelectToken={onOpenSelectToken('from')}
+        token={fromToken}
+        tokenBalance={balance}
+        hasInputError={insufficientBalance}
+        fiatValue={fiatValue}
       />
-      <Box
-        boxType='secondary'
+      <FlipTokensButton onClick={onClickFlipSwapTokens} />
+      <ToSection
+        getLocale={getLocale}
+        onClickSelectToken={onOpenSelectToken('to')}
+        token={toToken}
+        inputValue={toAmount}
+        onInputChange={handleOnSetToAmount}
+        hasInputError={false}
+        isLoading={isFetchingQuote}
```

Comment:
Is it possible to add a `disabled` prop here? Solana swaps do not support modifying the TO amount yet, so we should disable this input, and ideally also have a way to visually indicate that we don't support editting it.

Ok with doing it in a different PR if you think we need product-design feedback first.

Reply:
Added this prop!
Will ping design and see if they want to modify anything when `disabled`.